pyproject.toml

```
@@ -1,7 +1,55 @@
 [build-system]
-requires = ["setuptools", "lsst-versions"]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "lsst-felis"
+description = "Felis is a way of describing database catalogs, scientific and otherwise, in a language and DBMS agnostic way."
+license = {text = "BSD 3-Clause License"}
+readme = "README.md"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+keywords = ["lsst"]
+dependencies = [
+    "astropy",
+    "sqlalchemy >= 1.4",
+    "click",
+    "pyyaml",
+    "pyld",
+    "pydantic >= 2, < 3"
+]
+
+dynamic = ["version"]
+requires-python = ">=3.10.0"
```

```suggestion
requires-python = ">=3.11.0"
```
? Maybe it does work on 3.10 but you don't list it in the classifiers section.

---

I've changed this to use your suggested version.